.concourse/scripts/t-clippy-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+set -o pipefail
+
+GIT_DIR=$1
+
+cd $GIT_DIR
+
+# Fetch submodules.  Workaround for https://github.com/telia-oss/github-pr-resource/pull/200
+if [ ! -z "$SUBMODULE_ACCESS_TOKEN" ]; then
+    git submodule init
+    set +x
+    git config submodule.external/cryptography.url "https://${SUBMODULE_ACCESS_TOKEN}@github.com/PolymathNetwork/cryptography.git"
+    git config submodule.pallets/settlement.url    "https://${SUBMODULE_ACCESS_TOKEN}@github.com/PolymathNetwork/polymesh-settlement-module.git"
+    set -x
+    git submodule update pallets/settlement
+    git submodule update external/cryptography
+fi
+
+cargo +nightly clippy -- -A clippy::all -W clippy::complexity -W clippy::perf
```

Comment:
I believe this should be `cargo clippy`, not `cargo +nightly clippy` since the `nightly` (no date) toolchain is not installed but the versioned nightly is the default.

Reply:
PS: Here's the test run output:

```
+ set -o pipefail
+ GIT_DIR=githubpr-polymesh
+ cd githubpr-polymesh
+ '[' '!' -z 7fad027d973701e2bf0c554cb2a6ff9ce0def7db ']'
+ git submodule init
Submodule 'external/cryptography' (git@github.com:PolymathNetwork/cryptography.git) registered for path 'external/cryptography'
Submodule 'pallets/settlement' (git@github.com:PolymathNetwork/polymesh-settlement-module.git) registered for path 'pallets/settlement'
+ set +x
+ git submodule update pallets/settlement
Cloning into '/tmp/build/e0282143/githubpr-polymesh/pallets/settlement'...
Submodule path 'pallets/settlement': checked out 'b5f784292efa03829f1b9726d654c43883e5de8b'
+ git submodule update external/cryptography
Cloning into '/tmp/build/e0282143/githubpr-polymesh/external/cryptography'...
Submodule path 'external/cryptography': checked out 'dc249f0417168a67569000aefea2ed930b7cfd73'
+ cargo +nightly clippy -- -A clippy::all -W clippy::complexity -W clippy::perf
error: toolchain 'nightly-x86_64-unknown-linux-gnu' is not installed
```